test/test_essentials.sh

```diff
@@ -25,6 +25,18 @@ setUp() {
 #
 # Let the test begin
 #
+test_defining_private_variables() {
```

Comment:
In the current implementation, before the first test function there is a line of indentation from the comment. Please save this.